app/src/main/java/com/ataulm/whatsnext/api/LetterboxdApi.kt

```diff
@@ -14,25 +17,18 @@ interface LetterboxdApi {
             @Field("grant_type") grantType: String = "password"
     ): Single<AuthTokenApiResponse>
 
-    @FormUrlEncoded
-    @POST("auth/token")
-    fun refreshAuthToken(
```

Comment:
not part of the public API anymore, it's only necessary for the `RefreshAccessTokenInterceptor`